src/main/java/seedu/application/logic/commands/AddCommand.java

```diff
@@ -24,17 +24,17 @@ public class AddCommand extends Command {
         + PREFIX_COMPANY + "COMPANY "
         + PREFIX_ROLE + "ROLE\n"
         + "[Optional] "
-        + PREFIX_STATUS + "STATUS "
-        + PREFIX_DEADLINE + "DEADLINE \n\n"
+        + PREFIX_DEADLINE + "DEADLINE "
```

Comment:
Okay maybe we can stick to this order for the rest of our implementation as well